build.gradle.kts

```diff
@@ -15,4 +16,4 @@ tasks.wrapper {
     gradleVersion = "8.10.2"
     distributionUrl = "https://services.gradle.org/distributions/gradle-$gradleVersion-bin.zip"
     distributionSha256Sum = "31c55713e40233a8303827ceb42ca48a47267a0ad4bab9177123121e71524c26"
-}
\ No newline at end of file
+}
```

Comment:
Please leave the linefeed at the end of file, to make this and following diffs cleaner

Reply:
The auto-formatting added the newline here when touching the file. Just to clarify, are you asking to keep this newline or remove it? I'd personally keep it.

Reply:
I'm for having a newline at the end of each file. I probably misread the diff, it looked like the newline was removed.